### PR TITLE
added sivasai's intro

### PR DESCRIPTION
**Pull request by sivasaisilla**
Hey I have added my intro in readme file, hope my pr will be merged

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by sivasaisilla** [comment]
My intro is not showing in the readme file!! @Panquesito7

**Comment by Panquesito7** [comment]
> My intro is not showing in the readme file!! @Panquesito7

Your changes will be reflected once the pull request is approved by two or more maintainers and merged. 🙂

File: README.md [hunk 1]
```diff
@@ -29,3 +29,4 @@
...
 - Good Evening my name is Anubhav and I am an aspiring developer and a problem solver with expertise in various technologies. I have contributed to numerous projects and have gained recognition for my work. I consider myself fairly creative and a hardworking individual. I have a passion for technology and gaming. I have hands-on experience in building real-world projects and keep myself updated with all the news related to gaming and tech. When I’m not coding, you can find me playing games and exploring new technologies. - [Anubhav Adhikari](https://github.com/anubhav1206)
+- Hey Everyone, myself Sivasai and I am web developer with lot of enthusiasm to learn new things on the go. I know C++, Javascript, React, GoLang, MongoDB and Solidity. I am a noob in opensource. Check out my profile -[sivasaisilla](https://github.com/sivasaisilla)
```
Comment (KendallDoesCoding) [inline README.md]: I don't really like that you state that your a noob in opensource, that's not the right words and it makes you sound bad. Instead, let's put New To Open-Source :)!, what do you say?

Comment (sivasaisilla) [reply README.md]: Sure @KendallDoesCoding

**Pull request merged**